protocol.md

```diff
@@ -281,6 +281,24 @@ The steps to verify the *user* using OpenID Connect are as follows:
 
 The process to [register a new device](#device-registration) with an `OIDC` [Verification Method] is the same as described above. The only difference is that at the end of the process the *Tanker server* returns the *user*'s [Verification Key].
 
+### OpenID Connect with authorization code flow
+
+The *Tanker Server* is integrated with a list of trusted OpenID Connect provider. The *Tanker Server* uses its own `client ID` and `client secret` to act as a `service provider`. It follows the [the OpenID protocol](https://openid.net/specs/openid-connect-core-1_0.html#CodeFlowAuth).
+
+For the *application* to offer OpenID Connect as a verification method, *application* owners need to register the trusted OpenID Connect provider to the *Tanker App*.
```

Comment:
register to -> register with, I think?

Reply:
(also Verification Method is capitalized and linkified everywhere else, maybe we also want to do it here?)